cassandra-diagnostics-connector30/src/main/java/io/smartcat/cassandra/diagnostics/connector/NodeProbeWrapper.java

```diff
@@ -118,4 +118,14 @@ public long getRepairSessions() {
         return unreachableNodes;
     }
 
+    /**
+     * Get the information if the native transport is active on the node.
+     *
+     * @return true iff native transport is active
+     */
+    @Override
+    public boolean getNativeTransportActive() {
```

Comment:
Instead of providing a boolean for this value can this be wrapped into a class like nodeinfo or something that will have all the fields reported by nodetool info?